features/step_definitions/certification_steps.rb

```diff
@@ -0,0 +1,37 @@
+Then('I should be on the certification page') do
+  expect(@certification_page).to be_displayed
+end
+
+Then('certified by should be set to current user name') do
+  expect(@certification_page.certified_by.value).to eql(@current_user.name)
+end
+
+Then('certification date should be set to today') do
+  dd, mm, yyyy = Time.current.strftime('%d-%-m-%Y').split('-')
```

Comment:
I presume this should be;

```suggestion
  dd, mm, yyyy = Time.current.strftime('%d-%m-%Y').split('-')
```

Reply:
ooh yes. that would flicker on 1st-9th days otherwise i believe.

Reply:
tweaked